stdlib/Logging/src/Logging.jl

```diff
@@ -55,6 +90,18 @@ Alias for [`LogLevel(2000)`](@ref LogLevel).
 """
 const Error = Base.CoreLogging.Error
 
+"""
+    custom_log_levels::Dict{LogLevel,Tuple{String,Symbol}}
+
+Add to this dict to introduce a print style for a custom log.
+
+For instance:
+```
+custom_log_levels[LogLevel(-500)] = ("MyLog", :magenta)
+```
+"""
```

Comment:
```suggestion
"""

The normal way to do this is to use the [`@create_log_macro`](@ref) macro, which automatically adds to this dict, while also defining the corresponding macro with this name.
```

Reply:
Actually this was a left-over from an earlier design. It shouldn't be part of the API